internal/juno/juno.go

```diff
@@ -207,6 +209,8 @@ func starkNetJsonRPC(stateManager *state.Manager, blockManager *block.Manager,
 ) (*jsonrpc.JsonRpc, error) {
 	starkNetApi := starknet.New(stateManager, blockManager, transactionManager, synchronizer,
 		virtualMachine)
+	healthApi := health.New(synchronizer, virtualMachine)
+
```

Comment:
Why are `synchronizer` and `virtualMachine` the only ones determining the node's health?

Reply:
We have a couple of things to test:
- Blocks are stored properly
- The state is updating without problem
- We can call starknet_call and all the Python-related stuff
- Database is working properly

If we are syncing properly, that means that the blocks are stored properly and that the database is working. The only pending is the python-related stuff, which now is only the starknet_call, but soon we will add at also estimateFee.